app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="mobile.programming.musicapp">
+    package="mobile.programming.musicApp">
```

Comment:
Here, I have also changed the multi-name package name to camelcase.

Reply:
Good noting, I did not think about correct naming conventions when renaming the project for Firebase